template.json

```diff
@@ -1,69 +1,121 @@
 {
-    "Actor": {
-        "types": [
+    "Actor":
+    {
+        "types":
+        [
             "custos",
             "friendly",
             "antagonist",
             "fantasticalCreature"
         ],
-        "templates": {
-            "background": {
+        "templates":
+        {
+            "background":
+            {
                 "biography": ""
             },
-            "resources": {
-                "hitPoints": {
-                    "min": 0,
-                    "value": 10,
-                    "max": 40
+            "common":
+            {
+                "attributes":
+                {
+                    "hitPoints":
+                    {
+                        "min": 0,
+                        "value": 10,
+                        "max": 40
+                    },
+                    "pietas":
+                    {
+                        "min": 0,
+                        "value": 10,
+                        "max": 50
+                    },
+                    "npc": true
+                },
+                "peritiae":
+                {
+                    "deBello":
+                    {
+                        "value": 14,
+                        "specialties": {},
```

Comment:
Aren't specialties arrays? Didn't you have a ton of issues because of specialties becoming objects and couldn't that be the problem?

Reply:
It f**** up everything if I do and I have to add a lot of more css lines. So I maintain the margin-bottom that works fine

Reply:
Mhm I think you didn't answer to the right comment here